test/src/specs/tx_pool/conflict.rs

```diff
@@ -0,0 +1,95 @@
+use crate::utils::{blank, propose};
+use crate::{Net, Node, Spec};
+use ckb_types::bytes::Bytes;
+use ckb_types::core::{Capacity, TransactionView};
+use ckb_types::prelude::*;
+
+pub struct ConflictInPending;
+
+impl Spec for ConflictInPending {
+    crate::name!("conflict_in_pending");
+
+    fn run(&self, net: &mut Net) {
+        let node = &net.nodes[0];
+        let window = node.consensus().tx_proposal_window();
+        node.generate_blocks(window.farthest() as usize + 2);
+
+        let (txa, txb) = conflict_transactions(node);
+        node.submit_transaction(&txa);
+
+        node.submit_block(&propose(node, &[&txa]));
+        (0..window.closest()).for_each(|_| {
+            node.submit_block(&blank(node));
+        });
+        node.submit_transaction(&txb);
```

Comment:
I think the submit tx will return an error result here.

Reply:
Updated. PTAL.